service/src/main/java/uk/gov/hmcts/reform/fpl/validation/groups/ValidateCaseNumberGroup.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.reform.fpl.validation.groups;
```

Comment:
I think we renamed this to ValidateFamilymanNumberGroup in previous story no?

Reply:
yeah accidentally got remade during conflict resolution